modules/ROOT/pages/manage-backup-restore.adoc

```diff
@@ -4,6 +4,8 @@ Use the following procedures to back up and restore Runtime Fabric services and
 
 Data about the state of Runtime Fabric is distributed across the Kubernetes cluster. To preserve applications in the event of system failure, confirm that Runtime Fabric is automatically backed up. When scheduling backups, follow best practices by configuring hourly backups. Store your backup on external storage outside the Runtime Fabric cluster.
 
+Starting with `rtfctl` version 1.0.67, if you use an rtf-agent version higher than 2.6.51, the `rtfctl` backup bundle does not contain the Mule app deployments. The deployments are synched from the Runtime Manager UI at the time of restoration.
```

Comment:
The latest deployment configurations are synched from the Runtime Manager at the time of restoration